### PR TITLE
FIxing HTTP 406 error while runing the test suite (npm test)

### PR DESCRIPTION
**Pull request by Jacobojijo**
This is to fix issue #95 

## Solution

I modified the `scraping.js` file to use more browser-like User-Agent and Accept headers. Here are the key changes:

1. Added constants for user agent and accept header:

   ```javascript
   const userAgent = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36';
   const acceptHeader = 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8';
   
 * Created a `getWithHeaders` function to make requests with these headers:

    ```javascript
    function getWithHeaders(url) {
        return preq.get({
            uri: url,
            headers: {
                'User-Agent': userAgent,
                'Accept': acceptHeader
            }
        });
    }
    ```

* Updated all `preq.get()` calls to use `getWithHeaders()` instead.

* Modified the `meta()` function calls to include the headers:

    ```javascript
    return meta({
        uri: url,
        headers: {
            'User-Agent': userAgent,
            'Accept': acceptHeader
        }
    })
    ```
 This should improve the reliability of the test suite, especially when dealing with websites that have stricter requirements for incoming requests.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     "istanbul": "^0.4.5",
     "mocha": "^10.3.0",
     "mocha-jshint": "^2.3.1",
-    "mocha-lcov-reporter": "^1.3.0"
+    "mocha-lcov-reporter": "^1.3.0",
+     "nock": "^13.3.0"
```
Comment (mvolz) [inline package.json:17]: extra space here

File: test/scraping.js [hunk 2]
```diff
@@ -1,284 +1,149 @@
...
-			return meta.parseAll(chtml)
-			.then(function(results) {
-				Object.keys(results).forEach(function(metadataType) {
-					Object.keys(results[metadataType]).forEach(function(key) {
-						assert.notDeepEqual(results[metadataType][key], undefined); // Ensure all values are not undefined in response
-					});
-				});
-			});
-		});
-	});
```
Comment (mvolz) [inline test/scraping.js:282]: Stick to tabs please. (Probably this repo needs some linting added!)

**Review by mvolz — CHANGES_REQUESTED** [review]
Just some style stuff (we should add linting so that's automated, sorry!)

**Comment by Jacobojijo** [comment]
Hello @mvolz would you please do the PR review for the changes made ?

File: package.json [hunk 3]
```diff
@@ -51,4 +52,4 @@
     "url": "https://github.com/wikimedia/html-metadata/issues"
   },
   "homepage": "https://github.com/wikimedia/html-metadata"
-}
+}
```
Comment (mvolz) [inline package.json:55]: The red circle with the line through it means this file is missing a new line character. 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Review by mvolz — CHANGES_REQUESTED** [review]

**Comment by Jacobojijo** [comment]
@mvolz, can you review my PR? I also think a styling lint should be added to help in other future changes. I'm thinking of introducing one to this project. What do you think?

**Comment by Jacobojijo** [comment]
@mvolz, I guess this PR is done

**Pull request closed without merging**